src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService serviceInform = new UserService();
```

Comment:
```suggestion
        UserService userService = new UserService();
```
i am not saying that we should always name variables after their data types, but serviceInform is a bit unclear